<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeRestrictionsDialog</class>
 <widget class="QDialog" name="TimeRestrictionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>0</y>
     <width>401</width>
     <height>361</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="BlockPartsOfDoubleDays">
    <attribute name="title">
     <string>MW or TR</string>
    </attribute>
    <widget class="QFrame" name="verticalFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QTableWidget" name="MWandTRTable">
        <property name="styleSheet">
         <string notr="true">selection-color: rgb(170, 0, 0);</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <row>
         <property name="text">
          <string>07:45 - 09:00</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>09:10 - 10:25</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>10:35 - 11:50</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>11:45 - 12:55</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>13:10 - 14:25</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>14:35 - 15:50</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>16:00 - 17:15</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>17:45 - 19:00</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>19:10 - 20:25</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>20:35 - 21:50</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>MW</string>
         </property>
         <property name="background">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>TR</string>
         </property>
         <property name="background">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignLeft|AlignVCenter</set>
         </property>
         <property name="background">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="3" column="1">
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="4" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="5" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="5" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="6" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="6" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="7" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="7" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="8" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="8" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="9" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="9" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 0, 0);
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Blocked Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Time Available</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="BlockPartsOfDays">
    <attribute name="title">
     <string>Parts of Days</string>
    </attribute>
    <widget class="QFrame" name="verticalFrame1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>371</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QTableWidget" name="allDaysTable">
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <row>
         <property name="text">
          <string>07:45 - 08:35</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>08:45 - 09:35</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>09:45 - 10:35</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>10:45 - 11:35</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>11:45 - 12:35</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>M</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>T</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>W</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>R</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="1" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="1" column="4">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="2" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="2" column="4">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="3" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="3" column="2">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="3" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="3" column="4">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="4" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="4" column="2">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="4" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
        <item row="4" column="4">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 0, 0);
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Blocked Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Time Available</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="BlockWholeDays">
    <attribute name="title">
     <string>Whole Days</string>
    </attribute>
    <widget class="QFrame" name="formFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="headerText">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Days to Block&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="TuesBox">
        <property name="text">
         <string>Tuesday</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="WedBox">
        <property name="text">
         <string>Wednesday</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="ThursBox">
        <property name="text">
         <string>Thursday</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="FriBox">
        <property name="text">
         <string>Friday</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="MonBox">
        <property name="text">
         <string>Monday</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TimeRestrictionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TimeRestrictionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
